llvm/lib/IR/IRBuilder.cpp

```diff
@@ -644,38 +644,48 @@ CallInst *IRBuilderBase::CreateMaskedScatter(Value *Data, Value *Ptrs,
 /// Create a call to Masked Expand Load intrinsic
 /// \p Ty        - vector type to load
 /// \p Ptr       - base pointer for the load
+/// \p Align     - alignment for one element
```

Comment:
```suggestion
/// \p Align     - alignment of \p Ptr
```
I think? I don't think this alignment has to hold for all elements, just the base pointer.

Reply:
Agree, fixed